docs/learn/fundamentals/fees-resource-limits-metering.mdx

```diff
@@ -0,0 +1,212 @@
+
+### Find a transaction’s resource fee
+
+The best way to find the required resource fee for any smart contract transaction is to use the [`simulateTransaction` endpoint](../smart-contract-internals/contract-interactions/transaction-simulation) from the RPC, which enables you to send a preflight transaction that will return the necessary resource values and resource fee.
+
+### Resource limitations
+
+:::note
+
+Only smart contract transactions are subject to resource limitations.
+
+:::
+
+Stellar’s ledger close time is constrained to a few seconds, preventing the execution of arbitrarily large transactions, regardless of the resource fees involved. All resources mentioned in the prior section are subject to a per-transaction limit. A transaction’s memory (RAM) is also capped, though not subject to any charge.
+
+Resource limits are determined by a validator vote and can be adjusted based on network usage and ecosystem needs with a validator consensus.
+
+Find current resource limits in the [Resource Limits & Fees](../../reference/resource-limits-fees.mdx) page in the Reference section.
+
+## Inclusion fee
+
+The inclusion fee is the maximum bid (a bid denotes a dynamic fee, meaning it varies based on certain network conditions) the submitter is willing to pay for the transaction to be included in the ledger. The inclusion fee equals the number of operations in the transaction multiplied by the effective base fee for the given ledger:
+
+inclusion fee = # of operations \* effective base fee
+
+**Effective base fee:** the fee required per operation for a transaction to make it to the ledger. This cannot be lower than 100 stroops per operation (the network minimum).
+
+**Stroop:** the smallest unit of a lumen, one ten-millionth of a lumen (.0000001 XLM).
+
+:::note
+
+Transactions can have up to 100 operations per transaction except for transactions that execute a smart contract. Smart contract transactions are only allowed one operation per transaction (unless the transaction is getting [fee-bumped](../encyclopedia/fee-bump-transactions.mdx); this would add another operation), and the limits are instead specified in CPU instructions and other limits.
+
+:::
+
+When you set a base fee for a transaction, you are specifying the maximum amount you are willing to pay per operation in that transaction. This doesn’t necessarily mean you’ll pay that amount. You’ll only be charged the lowest amount needed for your transaction to make it to the ledger. If network traffic is light and the number of submitted operations or transactions is below the network ledger limit (configured by validators: currently 1,000 "classic" Stellar operations and 100 smart contract transactions), you will only pay the network minimum (configured by validators, currently 100 stroops).
```

Comment:
```suggestion
When you set a base fee for a transaction, you are specifying the maximum amount you are willing to pay per operation in that transaction. This doesn’t necessarily mean you’ll pay that amount. You’ll only be charged the lowest amount needed for your transaction to make it to the ledger. If network traffic is light and the number of submitted operations or transactions is below the network ledger limit (configured by validators: currently 1,000 non-smart-contract operations and 100 smart contract transactions), you will only pay the network minimum (configured by validators, currently 100 stroops).
```
I'm not sure of the best way to word this, but I think we're trying to avoid "classic" these days, right?

Reply:
You're right! I must have missed this one.